com.ibm.wala.cast.python.test/data/tf2_test_callbacks2.py

```diff
@@ -0,0 +1,11 @@
+import tensorflow as tf
+
+# Code from https://www.tensorflow.org/api_docs/python/tf/distribute/Strategy#example_usage_2
+strategy = tf.distribute.MirroredStrategy(["GPU:0", "GPU:1"])
+tensor_input = tf.constant(3.0)
+
+@tf.function
+def replica_fn(input):
```

Comment:
What is the expected result here? That `input` is a tensor?

Reply:
No, we want to check that the callback function is in the CG.

Reply:
Correct

Reply:
How many tensors are showing up?

Reply:
Since we don't have a pointer key for the function `replica_fn`, then we fail on https://github.com/ponder-lab/ML/blob/a19235b3bed5735ff0fd21b853b1b46cdedc358c/com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflowModel.java#L301

Reply:
But none of that mentions `replica_fn`. In other words, `replica_fn` should have one tensor parameter.